.github/workflows/test.yml

```diff
@@ -22,4 +28,17 @@ jobs:
           key: v1/${{ runner.os }}/node-${{ matrix.node }}/${{ hashFiles('**/yarn.lock') }}
           restore-keys: v1/${{ runner.os }}/node-${{ matrix.node }}/
       - run: yarn
+      - name: Set up @percy/cli master
+        if: ${{ github.event_name == 'workflow_dispatch' }}
+        run: |
+          cd /tmp
+          git clone --branch ${{ github.event.inputs.branch || 'master' }} --depth 1 https://github.com/percy/cli
```

Comment:
```suggestion
          git clone --branch ${{ github.event.inputs.branch }} --depth 1 https://github.com/percy/cli
```

Reply:
Lets keep this as a fallback

Reply:
we should remove this, as we're having `defaults` :)

Reply:
* add which - branch/commit/commit msg of CLI we're using for ease of navigation from SDK repo to CLI repo